### PR TITLE
[CAS-874] Using Style holder instead of serialising MessageListViewStyle

### PR DESCRIPTION
**Pull request by leandroBorgesFerreira**
[CAS-874](https://stream-io.atlassian.net/browse/CAS-874)

### 🎯 Goal

Fix the crash for serialising `MessageListViewStyle`

### 🛠 Implementation details

In this PR it is used an object to hold the Style for MessageLIstView, no need to serialise/deserialise the style object. 

### 🎨 UI Changes

No changes. 

### 🧪 Testing

Open message options and put the app onStop and onResume again. Observe that no crash happens, differently of the `develop` branch

### ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- ~[] Changelog is updated with client-facing changes~
- ~[ ] New code is covered by unit tests~
- ~[ ] Comparison screenshots added for visual changes~
- ~[ ] Affected documentation updated (CMS, cookbooks, tutorial)~
- [x] Reviewers added

### 🎉 GIF

giphy
_Fixing a crash_

### CODE REVIEW

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/style/StylesHolder.kt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package io.getstream.chat.android.ui.style
+
+import io.getstream.chat.android.ui.message.list.MessageListViewStyle
+
+internal object StylesHolder {
+    var messageListViewStyle: MessageListViewStyle? = null
```
Comment (JcMinarro) [inline stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/style/StylesHolder.kt:6]: Can't we have a non-nullable property here?

Comment (leandroBorgesFerreira) [reply stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/style/StylesHolder.kt:6]: 🤔  The way I see it to use `lateinit var`. What do your think?

Comment (leandroBorgesFerreira) [reply stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/style/StylesHolder.kt]: Done with static values

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/options/message/internal/MessageOptionsDialogFragment.kt [hunk 2]
```diff
@@ -335,36 +334,33 @@ internal class MessageOptionsDialogFragment : FullScreenDialogFragment() {
...
-            configuration: MessageOptionsView.Configuration,
+            configuration: MessageOptionsView.Configuration
         ): MessageOptionsDialogFragment {
             return MessageOptionsDialogFragment().apply {
                 arguments = bundleOf(
                     ARG_OPTIONS_MODE to optionsMode,
-                    ARG_OPTIONS_ITEM_STYLE to style,
                     ARG_OPTIONS_CONFIG to configuration
                 )
                 // pass message via static field
```
Comment (bychkovdmitry) [inline stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/options/message/internal/MessageOptionsDialogFragment.kt:366]: Maybe move `MessageOptionsDialogFragment.messageListViewStyle = messageListViewStyle` assignment here?

**Review by bychkovdmitry — APPROVED** [review]

**Pull request merged**